### PR TITLE
LTP: Fixed fremovexattr 01 and 02 tests

### PR DESCRIPTION
**Pull request by vamsikrishna935**
Tests were modified to use root file system.

### CODE REVIEW

File: tests/ltp/patches/ltp_fremovexattr01_fix.patch [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
++ // So modified the tests to use root file system.
+diff --git a/testcases/kernel/syscalls/fremovexattr/fremovexattr01.c b/testcases/kernel/syscalls/fremovexattr/fremovexattr01.c
+index a3ea1aa2b..9961c375d 100644
+--- a/testcases/kernel/syscalls/fremovexattr/fremovexattr01.c
++++ b/testcases/kernel/syscalls/fremovexattr/fremovexattr01.c
+@@ -70,10 +70,19 @@ static void cleanup(void)
+ {
+        if (fd > 0)
+                SAFE_CLOSE(fd);
++       remove(FNAME);
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr01_fix.patch:11]: use SAFE_REMOVE()

File: tests/ltp/patches/ltp_fremovexattr01_fix.patch [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
++       if (tst_umount(MNTPOINT) < 0) {
++               tst_res(TWARN | TERRNO, "umount device :mntpoint failed");
++       }
++       rmdir(MNTPOINT);
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr01_fix.patch]: SAFE_RMDIR()

File: tests/ltp/patches/ltp_fremovexattr01_fix.patch [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ }
+
+ static void setup(void)
+ {
++       rmdir(MNTPOINT);
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr01_fix.patch]: SAFE_RMDIR()

File: tests/ltp/patches/ltp_fremovexattr02_fix.patch [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
++//      .exp_err = EFAULT,
++//     },
+ };
+
+ static void verify_fremovexattr(unsigned int i)
+@@ -86,12 +86,19 @@ static void cleanup(void)
+ {
+        if (fd > 0)
+                SAFE_CLOSE(fd);
++       remove(FNAME);
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr02_fix.patch:24]: SAFE_REMOVE()

File: tests/ltp/patches/ltp_fremovexattr02_fix.patch [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
++       if (tst_umount(MNTPOINT) < 0) {
++               tst_res(TWARN | TERRNO, "umount device :mntpoint failed");
++       }
+++      rmdir(MNTPOINT);
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr02_fix.patch]: SAFE_RMDIR

File: tests/ltp/patches/ltp_fremovexattr02_fix.patch [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+ }
+
+ static void setup(void)
+ {
+        size_t i = 0;
+-
++       rmdir(MNTPOINT);
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr02_fix.patch]: SAFE_RMDIR

**Review by hukoyu — CHANGES_REQUESTED** [review]
Use SAFE_* frammework provided methods

File: tests/ltp/patches/ltp_fremovexattr02_fix.patch [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+--- a/testcases/kernel/syscalls/fremovexattr/fremovexattr02.c
++++ b/testcases/kernel/syscalls/fremovexattr/fremovexattr02.c
+@@ -55,9 +55,9 @@ struct test_case tc[] = {
+         .key = XATTR_TEST_KEY,
+         .exp_err = EBADF,
+         },
+-       {                               /* case 3: bad name attribute */
+-        .exp_err = EFAULT,
+-       },
++//     {                               /* case 3: bad name attribute */ TODO: Will be enabled once issue 169 is fixed
```
Comment (hukoyu) [inline tests/ltp/patches/ltp_fremovexattr02_fix.patch]: Change to https://github.com/lsds/sgx-lkl/issues/297

**Review by hukoyu — CHANGES_REQUESTED** [review]
Refer to issue https://github.com/lsds/sgx-lkl/issues/297 not 169

**Review by hukoyu — APPROVED** [review]
LGTM. cc @letmaik

**Review by letmaik — APPROVED** [review]

**Pull request merged**